src/commands/synthetics/command.ts

```diff
@@ -87,59 +87,7 @@ export class RunTestCommand extends Command {
       return 0
     }
 
-    return this.renderResults(results, summary, startTime)
-  }
-
-  private getAppBaseURL() {
-    return `https://${this.config.subdomain}.${this.config.datadogSite}/`
-  }
-
-  private renderResults(results: Result[], summary: Summary, startTime: number) {
-    // Rendering the results.
-    this.reporter?.reportStart({startTime})
-
-    if (!this.config.failOnTimeout) {
-      if (!summary.timedOut) {
-        summary.timedOut = 0
-      }
-    }
-
-    if (!this.config.failOnCriticalErrors) {
-      if (!summary.criticalErrors) {
-        summary.criticalErrors = 0
-      }
-    }
-
-    let hasSucceeded = true // Determine if all the tests have succeeded
-
-    const sortedResults = results.sort(this.sortResultsByOutcome())
-
-    for (const result of sortedResults) {
-      if (!this.config.failOnTimeout && result.timedOut) {
-        summary.timedOut++
-      }
-
-      if (result.result.unhealthy && !this.failOnCriticalErrors) {
-        summary.criticalErrors++
-      }
-
-      const resultOutcome = getResultOutcome(result)
-
-      if ([ResultOutcome.Passed, ResultOutcome.PassedNonBlocking].includes(resultOutcome)) {
-        summary.passed++
-      } else if (resultOutcome === ResultOutcome.FailedNonBlocking) {
-        summary.failedNonBlocking++
-      } else {
-        summary.failed++
-        hasSucceeded = false
-      }
-
-      this.reporter?.resultEnd(result, this.getAppBaseURL())
-    }
-
-    this.reporter?.runEnd(summary, this.getAppBaseURL())
-
-    return hasSucceeded ? 0 : 1
+    return renderResults(results, summary, this.config, startTime, this.reporter)
```

Comment:
And now for an aesthetic nit: Could we switch up the order here ? Either go with `startTime` before `this.config` or `this.config` and `this.reporter` before the `results, summary, startTime` ?

I'd also be cool with this switching to an object with alphabetical ordering, since five parameters is a lot.

Reply:
I went with the same order as the GitHub action: https://github.com/DataDog/synthetics-ci-github-action/blob/447c7c3f611e23f78381a8ad15d4664c7df3ce78/src/process-results.ts#L3-L11

Although, since we'll have to update the GH action anyway, we might as well follow your suggestion here 🙂 I'll push a commit with the changes!